tdrs-backend/tdpservice/users/static/admin/js/user_form_region_toggle.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener('DOMContentLoaded', function() {
+    const groupSelect = document.querySelector('#id_groups');
+    const regionFieldRow = document.querySelector('#id_regions').closest('.form-row, .field-box, .form-group');
+
+    function toggleRegionField() {
+        if (!groupSelect || !regionFieldRow) return;
+
+        const selectedOptions = Array.from(groupSelect.selectedOptions).map(opt => opt.textContent.trim());
+        const regionalRoles = ["OFA Regional Staff", "Developer", "Data Analyst"];
```

Comment:
Data Analyst will never be assigned regions.

```suggestion
        const regionalRoles = ["OFA Regional Staff", "Developer"];
```